central/controllers/referrals.go

```diff
@@ -21,7 +22,15 @@ func CreateReferral(c *gin.Context) {
 
 	// how many should we make?
 	count := 1
+	// how many times should we be able to use them
+	limit := 1
```

Comment:
takes a `limit` param in the api, defaults to 1 which will be incremented down to zero when the code is used